### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by vpodzime**

### CODE REVIEW

**Comment by vpodzime** [comment]
@cf-bottom jenkins with exotics, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/10740/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/10740/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-10740/

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/10748/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/10748/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-10748/

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/10755/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/10755/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-10755/

**Comment by vpodzime** [comment]
> Alright, I triggered a build:
> 
> [Build Status](https://ci.cfengine.com//job/pr-pipeline/10755/)

Retry without SUSE 15 (failing to spawn) and Ubuntu 16.04 hub (phased out): [Build Status](https://ci.cfengine.com/job/pr-pipeline/10767/)

File: deps-packaging/lmdb/distfiles [hunk 1]
```diff
@@ -1 +1 @@
-d35d4f6f46313d62fd342c9dcbf574432919ce5e802d2b6cbe2ebd549821e5c4  openldap-LMDB_0.9.31.tar.gz
+c0937223bba3c37c896809883a3c9b43049354578b239d6ed2476236a87f40c9  openldap-LMDB_0.9.32.tar.gz
```
Comment (craigcomstock) [inline deps-packaging/lmdb/distfiles:1]: @vpodzime I found in a nova PR on master that the sha for 0.9.31 changed! 🙀 

https://github.com/cfengine/buildscripts/pull/1418/files

**Comment by vpodzime** [comment]
> Ubuntu 16.04 hub (phased out)

Had to remove it from one more place, done now.

**Review by craigcomstock — APPROVED** [review]

**Review by larsewi — APPROVED** [review]
🚀

**Comment by vpodzime** [comment]
@cf-bottom jenkins with exotics, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/10820/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/10820/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-10820/

**Comment by vpodzime** [comment]
> Sure, I triggered a build:
> 
> [Build Status](https://ci.cfengine.com//job/pr-pipeline/10820/)
> 
> (with exotics)
> 
> **Jenkins:** https://ci.cfengine.com/job/pr-pipeline/10820/
> 
> **Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-10820/

Upgrade tests failed because we were not migrating PostgreSQL.

**Pull request merged**